docs/configuration/build_backend.md

```diff
@@ -0,0 +1,82 @@
+# The uv build backend
+
+!!! note
+
+    The uv build backend is currently in preview and may change in any future release.
+
+    By default, uv currently uses the hatchling build backend.
+
+A build backend transforms a source directory into a source distribution or a wheel. While uv
+supports all build backends (PEP 517), it ships with the `uv_build` backend that integrates tightly
+with uv.
+
+The uv build backend currently only supports Python code and only builds universal wheels. An
+alternative backend is required if you want to create a
+[library with extension modules](../concepts/projects/init.md#projects-with-extension-modules).
+
+To use the uv build backend, configure it in `pyproject.toml`:
+
+```toml
+[build-system]
+requires = ["uv_build>=0.6.13,<0.7"]
+build-backend = "uv_build"
+```
+
+You can also use `uv init` to generate a new project that uses the uv build backend:
+
+```shell
+uv init --build-backend uv
+```
+
+`uv_build` is a separate package from uv, optimized for a small size and high portability. `uv`
```

Comment:
```suggestion
`uv_build` is a separate package from uv, optimized for portability and small binary size. `uv`
```

Reply:
I wonder if we should omit the backticks from `uv` here? I don't have strong feelings. I would maybe say, "The `uv` package..." if you want to keep them.